docs/getting-started/quickstart.md

```diff
@@ -93,157 +117,89 @@ crate-type = ["cdylib"]
 
 [dependencies]
 substreams = "0.5.0"
+substreams-ethereum = "0.8.0"
 prost = "0.11"
+num-bigint = "0.4"
 
 [profile.release]
 lto = true
 opt-level = 's'
 strip = "debuginfo"
 ```
-{% endcode %}
 
-Install the Rust crate for the chain you want to use. The crates have the protobuf models for the specific chains and helper code. Run the `cargo` `add` command to obtain the Substreams crates and specify the name of the crate for the blockchain you want to use.
+### 3. Create protobufs
 
-Acquire the Ethereum crate for Substreams by using:
+Substreams modules are required to output protobuf encoded messages. The example protobuf definition from the [`substreams-ethereum-tutorial`](https://github.com/streamingfast/substreams-ethereum-tutorial) defines a field for the blockchain version. The value is set in the Substreams module handler.
+
+Copy and paste the content for the example protobuf definition into a new file named [`BasicExampleProtoData.proto`](https://github.com/streamingfast/substreams-ethereum-tutorial/blob/main/proto/basicexample.proto) and save it to a `proto` directory in the root, or main directory, of your Substreams module.
 
-```bash
-$ cargo add substreams-ethereum
 ```
+syntax = "proto3";
 
-Acquire the Solana crate for Substreams by using:
+package eth.basicexample.v1;
 
-```bash
-$ cargo add substreams-solana
+message BasicExampleProtoData {
+  int32 version = 1;
+}
 ```
 
-{% hint style="success" %}
-**Tip**: Crates are used when they are available for the chosen blockchain.\
-\
-Alternatively, generate the Rust structs from one of the chain-specific `spkg` packages, which contain the protobuf modules. [Read through the Rust crates documentation](../reference-and-specs/rust-crates.md) for details.
-{% endhint %}
+Use the `substreams protogen` command to generate the Rust code to communicate with the protobuf.
 
-## Quickstart examples overview
+```bash
+substreams protogen substreams-ethereum-tutorial.yaml
+```
 
-The relationships for the flow of data are defined in the Substreams manifest. Further information is available in the documentation for [defining complex data strategies](../reference-and-specs/manifests.md) through manifest files.
+### 4. Create Rust library and module handlers
 
-### Ethereum example
+Your Substreams module must contain a Rust library that houses the "module handlers." These handlers are responsible for handling blockchain data injected into the module at runtime. The example module handler extracts the version from the <i>Block</i> object in the `block.ver` property, logs it to the terminal, and assigns it to the `version` field of the [`BasicExampleProtoData`](https://github.com/streamingfast/substreams-ethereum-tutorial/blob/main/proto/basicexample.proto) protobuf.
```

Comment:
We should change the actual tutorial to output block's hash and block number. The `ver` should never be mentioned, it's kind of a detail of the version of the block, but not all our chains have a `ver`. While `hash` and `number` will work for any blockchain.

Can you make the necessary change in the tutorial code as well as in this document.

Reply:
Code updated in tutorial and pushed to its repo.